pcrmc/cli.py

```diff
@@ -79,6 +79,47 @@ def add_contact(name: List[str] = typer.Argument(...),
         )
 
 
+@app.command()
+def modify_contact(id: int = typer.Argument(...),
+                   field: str = typer.Option(str(), "--field", "-f"),
+                   value: str = typer.Option(str(), "--value", "-v")
+                   ) -> None:
+    """Modify contact by id."""
+    contacter = get_contacter()
+    _, error = contacter.modify_contact(id, field, value)
```

Comment:
We should retrieve some object here instead of _ which then gets reported to the user. See next comment

Reply:
done